editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -84,15 +109,17 @@ function getInteractionTargetFromEditorState(editor: EditorState): InteractionTa
 }
 
 function getApplicableStrategies(
-  strategies: Array<CanvasStrategy>,
+  metaStrategies: Array<MetaCanvasStrategy>,
   canvasState: InteractionCanvasState,
   interactionSession: InteractionSession | null,
   metadata: ElementInstanceMetadataMap,
   allElementProps: AllElementProps,
 ): Array<CanvasStrategy> {
-  return strategies.filter((strategy) => {
-    return strategy.isApplicable(canvasState, interactionSession, metadata, allElementProps)
-  })
+  return metaStrategies
+    .flatMap((ms) => ms(canvasState, interactionSession, metadata, allElementProps))
+    .filter((strategy) => {
+      return strategy.isApplicable(canvasState, interactionSession, metadata, allElementProps)
```

Comment:
probably checking whether a strategy is applicable can be rolled into the meta strategy as well?

Reply:
i mean to comment not to approve 🤦‍♂️